superset/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx

```diff
@@ -17,48 +18,65 @@ const propTypes = {
   form_data: PropTypes.object.isRequired,
   y_axis_zero: PropTypes.any,
   alert: PropTypes.string,
+  exploreState: PropTypes.object.isRequired,
 };
 
 class ControlPanelsContainer extends React.Component {
+  constructor() {
```

Comment:
i believe the constructor takes props in `constructor(props)`, and that `super` should be called with those props `super(props);`